### PR TITLE
Print ISO standard format in `IsoDuration#toString()`

### PR DESCRIPTION
**Pull request by zhenlineo**
Fixed a bug where the size of nanos should be `int`

### CODE REVIEW

File: driver/src/main/java/org/neo4j/driver/internal/InternalIsoDuration.java [hunk 1]
```diff
@@ -188,11 +195,65 @@ public int hashCode()
...
     public String toString()
     {
-        return "Duration{" +
-               "months=" + months +
-               ", days=" + days +
-               ", seconds=" + seconds +
-               ", nanoseconds=" + nanoseconds +
-               '}';
+        // print the duration in iso standard format.
+        if ( this == ZERO )
```
Comment (lutovich) [inline driver/src/main/java/org/neo4j/driver/internal/InternalIsoDuration.java]: Maybe `this.equals(ZERO)`? Then last `if` with "this was all zeros (but not ZERO for some reason)" comment can be removed.

**Review by lutovich — COMMENTED** [review]
@zhenlineo review completed. Made one comment. Would be also nice to add tests for zero and negative duration.
